webrtc/videoroom/config/runtime.exs

```diff
@@ -4,7 +4,8 @@ import Config
 # turn_servers: "addr:port:username:password:proto"
 config :membrane_videoroom_demo,
   stun_servers: System.get_env("STUN_SERVERS", "64.233.163.127:19302"),
-  turn_servers: System.get_env("TURN_SERVERS", "")
+  turn_servers: System.get_env("TURN_SERVERS", ""),
+  max_display_num: System.get_env("MAX_DISPLAY_NUM", "3")
```

Comment:
```suggestion
  max_display_num: System.get_env("MAX_DISPLAY_NUM", "3") |> String.to_integer()
```

At least the config will crash before application startup and we don't have to parse it later.

Reply:
This way we allow for passing non positive numbers. Not sure if we want to do this?

Reply:
Then we may want to do the checking in the config anyways? There is a lot of logic in those files anyway so putting the parsing in the config file would just simplify them.